### DISCUSSION TITLE
Any recommended ways to combine Torchvision pretrained models and MultiTaskModule?

### DISCUSSION THREAD

**Discussion by dsoselia** [ContinualAI/avalanche#716] [Q&A] [answered]
Seems somewhat hard since MultiTaskModule expects task id argument.

**Comment by AndreaCossu** [comment, marked as answer]
Hi @dsoselia ! What do you mean with combining? Your model can include a pretrained model inside, so that the forward of your whole model takes task id as argument but you don't forward it to the pretrained submodule. I think it depends on what you have to do here.

**Reply by AntonioCarta** [reply]
Basically, you have to create a wrapper for the original model. If your model is defined as a feature extractor+classifier it's really easy.
